src/metabase/pulse/render/body.clj

```diff
@@ -610,8 +609,7 @@
 (defn- multiple-scalar-series
   [joined-rows _x-cols _y-cols _viz-settings]
   [(for [[row-val] (map vector joined-rows)]
-     {:name          nil
```

Comment:
`name` was meant for a custom metric or breakout column name, but apparently since for series with breakout columns need their keys to be calculated on FE anyway, it's better to also calculate series keys for series with one dimension also. That's why `name` is removed.